source/installation/kafka-deployments/install-docker.txt

```diff
@@ -15,3 +15,77 @@ Install with a Containerized Cluster Using Docker
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka in a containerized environment 
+using `Docker <https://docs.docker.com/engine/>`__. This method configures Kafka 
+to store data locally in Docker containers. Use this method if you want to learn 
+how to configure your own multi-server Kafka environment. 
+
+.. warning::
+
+   This deployment method is not recommended for production workloads. 
+
+About this Task
+---------------
+
+`Docker volumes <https://docs.docker.com/storage/volumes/>`__ can be used for 
+persisting data if a container stops running or is restarted. 
+
+Before you Begin
+----------------
+
+- You must have Docker installed on your computer. For more information, see  
+  `Install Docker Engine <https://docs.docker.com/engine/install/>`__. 
+
+- Download the following example ``docker-compose`` that configures a Kafka 
+  cluster and Kafka Connect cluster to run Relational Migator locally: Kafka 
+  Docker Compose. 
+
+Steps
```

Comment:
These steps don't match the compose file. The steps should basically be:
1. Download the compose file from the Download Center
2. Configure environment variables
2.1 `MIGRATOR_PATH_DATA` for persistence of Relational Migrator data
2.2 (Optional) `MIGRATOR_PATH_DRIVER` to point to a .jar file for additional JDBC drivers (e.g. Oracle/MySQL)
3. Pull down the Docker images using `docker-compose -f docker-compose-migrator-kafka.yml pull`
4. Start the Docker stack using `docker-compose -f docker-compose-migrator-kafka.yml up`

@alwaysbemark Can you please check and confirm the instructions?

Reply:
Updated.